tests/optim/test_single_node_adascale.py

```diff
@@ -113,15 +112,15 @@ def test_grad_accum(test_case, cpu):
         optim.zero_grad()
 
 
-@skip_if_no_gpu
+@skip_if_no_cuda
```

Comment:
Thanks for updating this! You will need to merge with my commit just went in. Sorry about the conflict.

Reply:
no problem !